pkg/lib/v0_2_0/getaction.go

```diff
@@ -0,0 +1,17 @@
+package v0_2_0
+
+type GetActionTriggeredEventData struct {
+	EventData
+	ProblemDetails ProblemDetails `json:"problemDetails"`
```

Comment:
we need to change that to `problem` instead of `problemDetails`, to be consistent with the name of the property in the `RemediationTriggeredEventData` struct